lib/commands/view.js

```diff
@@ -0,0 +1,51 @@
+'use strict'
+
+const npmlog = require('npmlog')
+const libnpm = require('libnpm')
+
+const { h, renderToString } = require('ink')
+const { PackageView, PackageFields } = require('../components/view.jsx')
+
+const View = module.exports = {
+  command: 'view [<pkg>[@<version>]] [<field>...]',
+  aliases: ['v', 'info', 'show'],
+  describe: 'Show information about a package',
+  builder (yargs) {
+    return yargs.help().alias('help', 'h').options(View.options)
+  },
+  options: Object.assign(require('../common-opts'), {}),
```

Comment:
Wouldn't it make more sense if these arguments were reversed?

Reply:
Yep, don't know why I did it that way... Not that it matters, at the moment.